draft-ietf-tls-rfc8446bis.md

```diff
@@ -5544,7 +5544,11 @@ Forward secret with respect to long-term keys:
   the handshake is complete, this does not compromise the security of the
   session key (see {{?DOW92=DOI.10.1007/BF00124891}}), as long as the session key itself has
   been erased. The forward secrecy property is not satisfied
```

Comment:
```suggestion
  session key (see {{?DOW92=DOI.10.1007/BF00124891}}), as long as the session key itself (and all material that could be used to recreate the session key) has
  been erased. In particular, the key shares, shared secrets, and keys derived in the TLS Key Schedule other than `binder_key`, `resumption_secret`, and PSKs derived from the `resumption_Secret` also need to be erased.  The forward secrecy property is not satisfied
```

Reply:
You don't need to erase the shares, you need to erase the secrets from which those shares were generated (I don't know a name that would allow you to concisely identify those).

Reply:
Changed to "private keys related to key shares". I think "private keys related to key shares, shared secrets" cover everything.

For DH you need to delete both private keys and the resulting shared secret. In a KEM you need to delete one private key and the shared secret.